### PR TITLE
update vllm_hpu_extension commit to 24039a3

### PR DESCRIPTION
**Pull request by ccrhx4**
to fix part of the https://github.com/HabanaAI/vllm-fork/issues/443

update the vllm_hpu_extension commit id to https://github.com/HabanaAI/vllm-hpu-extension/pull/26/commits/24039a3e0abc9f1b90345018ccab5c711e6c042a

### CODE REVIEW

File: requirements-hpu.txt [hunk 1]
```diff
@@ -8,5 +8,5 @@ pandas
 tabulate
 setuptools>=61
 setuptools-scm>=8
-vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@0063520
+vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@24039a3
```
Comment (michalkuligowski) [inline requirements-hpu.txt:11]: Shouldnt it be 4e911b4 ?

**Review by michalkuligowski — CHANGES_REQUESTED** [review]

**Review by madamczykhabana — CHANGES_REQUESTED** [review]
Revert for PR26 is pending: https://github.com/HabanaAI/vllm-hpu-extension/pull/31

**Comment by michalkuligowski** [comment]
This needs to be rejected due to: https://github.com/HabanaAI/vllm-hpu-extension/pull/31

**Comment by ccrhx4** [comment]
Inference accuracy is at the risk of being compromised when the softmax_mode == "fast".

From the Habana document, https://docs.habana.ai/en/latest/PyTorch/Model_Optimization_PyTorch/Optimization_in_PyTorch_Models.html#using-fused-sdpa

> Using fast Softmax may affect inference accuracy.
> Only BF16 data type is supported with fast Softmax.
> Fast Softmax is not supported when running training in recompute mode with is_causal = False.

Even in the optimum-habana, the softmax_mode is default as "None". https://github.com/huggingface/optimum-habana/blob/f488ab66329b2c3a46063292b2822c1cab068756/optimum/habana/transformers/generation/configuration_utils.py#L38

Please reconsider this issue, I understand the performance is important, but the model needs to be performed correctly as it has been trained to.

**Comment by michalkuligowski** [comment]
Please retest on latest versions and open a new PR if needed
